.github/workflows/tests.yaml

```diff
@@ -6,6 +6,20 @@ on:
       - main
 
 jobs:
+  lint:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v4
+      - uses: pnpm/action-setup@v4
+        with:
+          version: 9
```

Comment:
This will cause an error together with the other change where we've set the "packageManager" to pnpm 10 in package.json (due to version mismatch).

Reply:
Currently this PR is going to merge into #66, will need to rebase and resolve that YAML file manually.

Reply:
Updated, should work now.